openedx/core/lib/api/permissions.py

```diff
@@ -80,3 +86,64 @@ def has_object_permission(self, request, view, obj):
         return (request.user.is_staff or
                 CourseStaffRole(obj.course_id).has_user(request.user) or
                 request.method in permissions.SAFE_METHODS)
+
+
+def get_profile_visibility(user_profile, user, configuration=None):
```

Comment:
The methods were originally in openedx/core/djangoapps/user_api/accounts/serializers.py. If they are exposed outside of user_api, they should be moved into openedx/core/djangoapps/user_api/accounts/api.py.